docker/compose.testing.yaml

```
@@ -0,0 +1,121 @@
+    networks:
+      - minio_test
+    # https://docs.docker.com/reference/compose-file/services/#profiles
+    profiles:
+      - minio_test
+    # https://docs.docker.com/reference/compose-file/services/#restart
+    restart: unless-stopped
+  postgres-test:
+    # https://docs.docker.com/compose/environment-variables/
+    # https://docs.docker.com/reference/compose-file/services/#environment
+    environment:
+        # https://github.com/docker-library/docs/blob/master/postgres/README.md#postgres_db
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)